src/applications/edu-benefits/1995/submitForm.js

```diff
@@ -0,0 +1,23 @@
+import { submitToUrl } from 'platform/forms-system/src/js/actions';
+import { transformForSubmit } from 'platform/forms-system/src/js/helpers';
+
+const submitForm = (form, formConfig) => {
+  const body = formConfig.transformForSubmit
+    ? formConfig.transformForSubmit(formConfig, form)
+    : transformForSubmit(formConfig, form);
+
+  const eventData = {
```

Comment:
We have a helper for recording events [here](https://github.com/department-of-veterans-affairs/vets-website/blob/1a57f2caeae2c22816e1ca4fa2a973a35c53dbd5/src/platform/monitoring/record-event.js#L21). Would it be possible to use this instead of changing the forms-system. 

Updating this form-system action imposes the burden of regression testing all of the forms on the site (the tracking behavior) which isn't covered by our automated tests.

Reply:
just to clarify- we have automated testing on all of the forms but it doesn't cover the GA behavior yet.

Reply:
We need the ability to record additional data as part of the <form_id>-submission-successful event that is recorded by `submitToUrl`.

Duplicate the `submitToUrl` functionality in the 1995 submit handler doesn't seem like a good solution.  Do you have any suggestions?

Reply:
Thanks for the additional context. this should work then.